files/default/sql/ddl/3.0.0__initial_tables.sql

```diff
@@ -1392,6 +1392,7 @@ CREATE TABLE `on_demand_feature` (
   `primary_column` tinyint(1) NOT NULL DEFAULT '0',
   `description` varchar(10000) COLLATE latin1_general_cs,
   `type` varchar(1000) COLLATE latin1_general_cs NOT NULL,
+  `idx` int(11) NOT NULL DEFAULT 0,
```

Comment:
Why do we need a new column for ordering the features? We can use the`id` which is automatically incremented.